realm/realm-jni/build.gradle

```diff
@@ -83,7 +83,21 @@ def allTargets = [
     new Target( name:'x86_64',  abi:'x86_64',      toolchain:toolchains.find {it.name == 'x86_64'}, cflags:[] )
 ]
 
-def targets = ext.buildTargetAbis == null ? allTargets : allTargets.findAll {ext.buildTargetAbis.contains(it.abi)}
+def targets
+if (ext.buildTargetAbis == null) {
+    targets = allTargets;
+} else {
+    targets = ext.buildTargetAbis.collect {
+        def targetAbi = it
```

Comment:
why introducing this variable? if you want `it` to be renamed you can just use `collect { targetAbi -> ... }`